verilog/transform/obfuscate.h

```diff
@@ -23,6 +23,12 @@
 
 namespace verilog {
 
+// Collect all identifiers under module header subtree in the CTS.
+// This could be useful when interface names are required to be
+// preserved.
+absl::Status CollectInterfaceNames(absl::string_view content,
+                                   verible::StringSet* if_names);
```

Comment:
I'd just spell out `std::set<std::string>` to make it clear without having to lookup a typedef.